custom/demographics_print.php

```diff
@@ -0,0 +1,593 @@
+<?php
+// Copyright (C) 2009-2016 Rod Roark <rod@sunsetsystems.com>
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+
+// This will print a blank form, and if "patientid" is specified then
+// any existing data for the specified patient is included.
+//
+// Reduced and simplified by Sam Johnson (2011-03-19) for version that meets MSI needs.
+
+// Modified by Rod to be invoked from the custom/ directory by the normal
+// demographics_print.php script when MSI Rapid Workflow is activated.
+
+//The code below is copied from the original report, and fetches each data item in the demographic layout (if not 'unused')
+
+// require_once("../interface/globals.php");
```

Comment:
Isn't this a security issue?

Reply:
Or are you using $encounter to control this (since I think there is no way to run this script unless it's included by another script that has the $encounter value set)?

Reply:
Actually this entire file should be omitted. I just realized it covers an obsolete case and will never even be invoked.